rules/linux/persistence_kernel_module_activity.toml

```diff
@@ -1,7 +1,8 @@
 [metadata]
 creation_date = "2020/02/18"
-maturity = "production"
-updated_date = "2021/03/03"
+deprecation_date = "2021/04/15"
+maturity = "deprecated"
```

Comment:
could user, command line, or parent information help narrow the scope of this one?

Reply:
We have a more robust endpoint rule for this, which I'd love to see the results of to resurrect this one.